File: app/src/main/java/ae/innovativesolutions/wisdomhours/data/DataManager.java


package ae.innovativesolutions.wisdomhours.data;

import ae.innovativesolutions.wisdomhours.data.local.db.DbHelper;
import ae.innovativesolutions.wisdomhours.data.local.prefs.PreferencesHelper;
import ae.innovativesolutions.wisdomhours.data.remote.ApiHelper;

/**
 * Created by Zuhaib Ahmad on 07/07/17.
 */

public interface DataManager extends DbHelper, PreferencesHelper, ApiHelper {
}

File: app/src/main/java/com/zuhaibahmad/template/data/DataManager.kt


package com.zuhaibahmad.template.data

import com.zuhaibahmad.template.data.local.db.DbHelper
import com.zuhaibahmad.template.data.local.prefs.PreferencesHelper
import com.zuhaibahmad.template.data.remote.ApiHelper
import com.zuhaibahmad.template.utils.OpenClass
import io.reactivex.Completable

/**
 * Created by Zuhaib on 9/17/2017.
 */
@OpenClass
interface DataManager : DbHelper, PreferencesHelper, ApiHelper {
}
